compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/platform/SkiaLayerTest.kt

```diff
@@ -254,29 +259,31 @@ class SkiaLayerTest {
             translationX = 60f,
             translationY = 7f,
             rotationX = 45f,
-            transformOrigin = TransformOrigin(0f, 0f)
+            transformOrigin = TransformOrigin(0f, 0f),
+            cameraDistance = Float.MAX_VALUE
         )
         val matrix = layer.matrix
 
         val y = (10 * cos45).roundToInt()
-        val translationY = (7 * cos45).roundToInt()
+        val translationY = 7
```

Comment:
Why this changes?
It means that the previous value was a workaround to correct a wrong transformation?

Reply:
Yes. Note that `transformOrigin` is `(0, 0)`, so rotation around it should transfer this point.